### PR TITLE
Update info re. VPN

### PR DESCRIPTION
**Pull request by Jazzagi**
Add explanation around how the VPN disallows device-to-device traffic

Change-type: patch

---

> *Please make sure to read the [CONTRIBUTING](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md) document before opening the PR for relevant information on contributing to the documentation. Thanks!*

### CODE REVIEW

File: pages/learn/welcome/security.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Both the Docker pull request and the actual image download process are performed
 
 When the VPN is enabled, SSH access is available to the service using the {{ $names.company.lower }} dashboard or the CLI.
 
-The {{ $names.company.lower }} VPN disallows device-to-device traffic and prohibits outbound traffic to the Internet.  If a device were compromised, this ensures that it cannot contaminate another device.
+The {{ $names.company.lower }} VPN disallows device-to-device traffic and prohibits outbound traffic to the Internet.  If a device were compromised, this ensures that it cannot contaminate another device. To achieve this the VPN service is configured to run with iptables default `FORWARD` policy set to `DROP` and we do not set OpenVPN `--client-to-client` config option server side, so there is no physical way for the traffic between clients to traverse the interface(s).
```
Comment (klutchell) [inline pages/learn/welcome/security.md:53]:
```suggestion
The {{ $names.company.lower }} VPN disallows device-to-device traffic and prohibits outbound traffic to the Internet.  If a device were compromised, this ensures that it cannot contaminate another device. To achieve this the VPN service is configured to run with iptables default `FORWARD` policy set to `DROP` and we do not set OpenVPN `--client-to-client` config option server side, so there is no way for the traffic between clients to traverse the interface(s). 
```
I hesitate to use the word "physical" in this context, as they are still physically on the same cluster.

Comment (Jazzagi) [reply pages/learn/welcome/security.md:53]: Sounds sensible

Comment (vipulgupta2048) [reply pages/learn/welcome/security.md:53]: Is there some docs we can link here to show what option this is?

**Comment by Jazzagi** [comment]
> Thanks for making this PR @Jazzagi Great job 🎉🎉🎉

Thanks. Any reason we can't merge this now? Seems to be sitting for a long time for such a small change

@vipulgupta2048

**Comment by klutchell** [comment]
Sorry @Jazzagi , I think we were waiting on you to respond or commit to the review suggestions. I can take over this PR or we can have a chat about next steps?

**Review by vipulgupta2048 — APPROVED** [review]
Houston, you are clear for takeoff

**Pull request merged**